packages/react-client/src/hooks/useScreenShare.ts

```diff
@@ -74,11 +96,12 @@ export const useScreenShare = (): ScreenshareApi => {
 
     await Promise.all(removeTrackPromises);
 
-    setState(null);
-  }, [state, fishjamClientRef, setState]);
+    cleanMiddleware();
```

Comment:
In this implementation, we don't have the ability to `stopStreaming` without stopping the screen capturer. This is possible for the microphone and camera. Is this intended behaviour? If so, we could create another ticket for this.

Reply:
So far this is an intended behavior. Stopping streaming without stopping the capturer will result in browser telling the end user he is in fact sharing his tab.